### PR TITLE
朱道冰Week1

### PR DESCRIPTION
**Pull request by IcePigZDB**
### What problem does this PR solve?
朱道冰 week1 work.

### What have you done or changed in this PR?
朱道冰 week1 work.
### What test did you complete before submitting the PR?
朱道冰 week1 work.

### CODE REVIEW

File: Week1-Database-Introduction/DaoBingZhu-Week1/ExperimentReport.md [hunk 1]
```diff
@@ -0,0 +1,483 @@
...
+   (integer) 1
+   > pfcount hll
+   (integer) 4
+   ```
+
+ ## 特点：
+
+* Redis keys are binary safe, this means that you can use any binary sequence as a key, from a string like "foo" to the content of a JPEG file. The empty string is also a valid key.The maximum allowed key size is 512 MB.
+* Automatic creation and removal of keys applies to all the Redis data types composed of multiple elements -- Streams, Sets, Sorted Sets and Hashes.
+* 最大的收获是熟悉了redis一部分命令文档，以后有不懂的可以直接差redis文档了。
```
Comment (ehds) [inline Week1-Database-Introduction/DaoBingZhu-Week1/ExperimentReport.md:354]: 差 -> 查

Comment (IcePigZDB) [reply Week1-Database-Introduction/DaoBingZhu-Week1/ExperimentReport.md]: fixed thx

File: Week1-Database-Introduction/DaoBingZhu-Week1/ExperimentReport.md [hunk 2]
```diff
@@ -0,0 +1,483 @@
...
+     # initialize
+     $ mysql_install_db --user=mysql --basedir=/usr --datadir=/var/lib/
+     $ mysql_secure_installation
+     # 开启服务这个每个数据库都需要
+     $ systemctl start mariadb
+   ```
+
+ * 安装MongoDB
+
+ * 安装j
```
[the remark this answers is not in the document]
Comment (IcePigZDB) [reply Week1-Database-Introduction/DaoBingZhu-Week1/ExperimentReport.md]: fixed thx.

**Comment by karmanluo** [comment]
LGTM

**Pull request merged**